wagtail/admin/panels.py

```diff
@@ -417,13 +417,15 @@ class PanelGroup(Panel):
     Concrete subclasses must attach a 'children' property
     """
 
-    def __init__(self, children=(), *args, **kwargs):
+    def __init__(self, children=(), permission=None, *args, **kwargs):
```

Comment:
This usage creates a conflict with the positional args on `Panel`, for now we could use kwargs.pop here.

```
    def __init__(self, children=(), *args, **kwargs):
        permission = kwargs.pop('permission', None)
```